generator/mavschema.xsd

```diff
@@ -230,6 +230,9 @@
     </xs:complexType>
 </xs:element>
 
+<!-- definition entry elements attributes (like the ones used on MAV_CMD for example) -->
```

Comment:
Should this go up near the top where all of the attributes are defined?

Reply:
I kind of copy-past how it done before, I didn't want to change too much